src/Models/CohortsModel.php

```diff
@@ -53,4 +53,12 @@ public function getAll(): array
 
         return $cohorts;
     }
+
+    public function getDates(): array
+    {
+        $query = $this->db->prepare("SELECT `id`, `date` FROM `cohorts`;");
+
+        $query->execute();
+        return $query->fetchAll();
+    }
```

Comment:
Seems really similar to something you did on story 1?

Reply:
done!